core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -340,4 +340,8 @@ private TableProperties() {
 
   public static final String UPSERT_ENABLED = "write.upsert.enabled";
   public static final boolean UPSERT_ENABLED_DEFAULT = false;
+
+  public static final String POS_DELETE_HAS_SAME_SEQ_WITH_REFS_ENABLED =
+      "write.pos-delete-has-same-seq-with-refs.enabled";
```

Comment:
I haven't figured out a shorter name.

Reply:
Can we turn it on automatically by conditions like whether it's upsert or not?